modules/tests/shared/src/test/scala/lucuma/core/model/SiderealTrackingSuite.scala

```diff
@@ -52,7 +52,7 @@ final class SiderealTrackingSuite extends DisciplineSuite {
     val pmdec = ProperMotion.Dec.milliarcsecondsPerYear.reverseGet(BigDecimal(938.527))
     val tracking = SiderealTracking(coord, Epoch.J2000, ProperMotion(pmra, pmdec).some, none, Parallax.fromMicroarcseconds(203887).some)
     val refEpoch = Instant.ofEpochSecond(4102444800L)
-    assertEquals(tracking.at(refEpoch), Coordinates.fromHmsDms.getOption("11 04 48.043284 +43 33 09.795210"))
+    assertEquals(tracking.at(refEpoch), Coordinates.fromHmsDms.getOption("11 04 48.043689 +43 33 09.794281"))
```

Comment:
the difference is very small, I assume given by the leap seconds right? should we validate this?